common/religions/secular_group.txt

```diff
@@ -268,6 +268,72 @@ secular_group = {
 		feminist = yes
 		matrilineal_marriages = yes
 
+		dynamic_cult = no	# Warcraft - can't handle it yet
+	}
+	suntouched = {
+		graphical_culture = muslimgfx
+		secondary_event_pictures = arcane_religion
+
+		icon = 74
+		heresy_icon = 75
+
+		color = { 1 0.85 0 }
+		parent = mystery_of_the_makers
+
+		scripture_name = THE_ENCYCLOPEDIA
+		priest_title = SUN_PRIEST
+		high_god_name = GOD_AMANTHUL
+		god_names = {
+			GOD_AMANTHUL GOD_EONAR GOD_NORGANNON GOD_GOLGANNETH GOD_KHAZGOROTH
+		}
+		evil_god_names = {
+			THE_VOID
+		}
+
+		# Intermarries
+		intermarry = secular_group
+
+		# Modifiers
+		character_modifier = {
+			# Bonuses
+			martial = 1
+			build_cost_temple_modifier = -0.2
+			land_morale = 0.1
+			
+			# Maluses
+			diplomacy = -1
+			plot_power_modifier = -0.1
+			siege_speed = -0.1
+			
+			# Opinions of other religions
+			general_opinion = -25
+			same_religion_opinion = 25
+			rel_head_opinion = 25
+			burning_legion_religion_opinion = -175
+			death_god_opinion = -175
+			old_gods_worship_opinion  = -175
+		}
+
+		# Secular laws
+		can_demand_religious_conversion = no
```

Comment:
Are you sure about this?
Others secular religions can't convert, but they also have vassal opinion bonus to show pluralism.

Reply:
forgot about these